runners/spark/src/test/java/org/apache/beam/runners/spark/DeDupTest.java

```diff
@@ -53,7 +53,6 @@ public void testRun() throws Exception {
 
     PAssert.that(output).containsInAnyOrder(EXPECTED_SET);
 
-    EvaluationResult res = (EvaluationResult) p.run();
-    res.close();
+    p.run();
```

Comment:
If I recall correctly, it is now `TestPipeline` that makes it so `run()` is blocking and terminating (for streaming cases, watching the counts of successes/failures and timing out). In the newly proposed APIs this may return immediately before anything is run.